spec/models/nationalisation_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+describe Nationalisation do
+  describe "associations" do
+    it { should belong_to(:trainee) }
```

Comment:
We're favouring `expect` over `should` https://www.betterspecs.org/#expect. Rspec should be enforcing this 🤷 https://github.com/DFE-Digital/register-trainee-teacher-data/blob/master/spec/spec_helper.rb#L11

Reply:
done https://github.com/DFE-Digital/register-trainee-teacher-data/pull/60/commits/1559281af097967dd3468045f676caacb6668304